integration-tests/tests/integration.rs

```diff
@@ -0,0 +1,218 @@
+
+        let work_dir = TempDir::new()?;
+        let resource_dir = TempDir::new()?;
+        let policy_path = NamedTempFile::new()?;
+
+        let mut auth_pubkey_path = NamedTempFile::new()?;
+        auth_pubkey_path.write(auth_pubkey.as_bytes())?;
+
+        let attestation_token_config = match &test_parameters.attestation_token_type[..] {
+            "Ear" => AttestationTokenConfig::Ear(ear_broker::Configuration {
+                duration_min: 5,
+                ..Default::default()
+            }),
+            "Simple" => AttestationTokenConfig::Simple(simple::Configuration::default()),
+            _ => bail!("Unknown attestation token type. Must be Simple or Ear"),
+        };
+
+        let kbs_config = KbsConfig {
+            attestation_token: AttestationTokenVerifierConfig {
+                trusted_certs_paths: vec![],
+                insecure_key: true,
+                trusted_jwk_sets: vec![],
+                extra_teekey_paths: vec![],
+            },
+            attestation_service: AttestationConfig {
+                attestation_service: AttestationServiceConfig::CoCoASBuiltIn(Config {
+                    work_dir: work_dir.path().to_path_buf(),
+                    rvps_config: RvpsConfig::BuiltIn(RvpsCrateConfig::default()),
+                    attestation_token_broker: attestation_token_config,
+                }),
+                timeout: 5,
+            },
+            http_server: HttpServerConfig {
+                sockets: vec!["127.0.0.1:8080".parse()?],
+                private_key: None,
+                certificate: None,
+                insecure_http: true,
+            },
+            admin: AdminConfig {
+                auth_public_key: None,
+                insecure_api: true,
+            },
+            policy_engine: PolicyEngineConfig {
+                policy_path: policy_path.path().to_path_buf(),
+            },
+            plugins: vec![PluginsConfig::ResourceStorage(RepositoryConfig::LocalFs(
+                LocalFsRepoDesc {
+                    dir_path: resource_dir.path().to_str().unwrap().to_string(),
+                },
+            ))],
+        };
+
+        Ok(TestHarness {
+            kbs_config,
+            auth_privkey,
+            _test_parameters: test_parameters,
+        })
+    }
+
+    async fn run(&self) -> Result<()> {
+        let api_server = ApiServer::new(self.kbs_config.clone()).await?;
+
+        let kbs_server = api_server.server()?;
+        let kbs_handle = kbs_server.handle();
+
+        actix_web::rt::spawn(kbs_server);
+
+        self.wait().await;
+        self.set_secret(SECRET_PATH.to_string(), SECRET_BYTES.as_ref().to_vec())
+            .await?;
+        self.set_policy(PolicyType::AllowAll).await?;
+
+        let secret = self.get_secret(SECRET_PATH.to_string()).await?;
+
+        assert_eq!(secret, SECRET_BYTES);
+        info!("TEST: test completed succesfully");
+
+        kbs_handle.stop(true).await;
```

Comment:
Here we don't need to explicitly kill KBS task, and we can simplify 136~139 lines to
```rust
actix_web::rt::spawn(async { api_server.serve().await });
```
This can avoid commit https://github.com/confidential-containers/trustee/pull/619/commits/c65676cb383115fc3785615eb0f545bbee29d72b. Because the `run` function is only called once in each test case, and each test case has another sub-`#[actix_rt:test]` macro, which is equivalent to creating a sub-asynchronous runtime, the whole sub-runtime will be automatically destroyed after the test case ends, which means that the KBS coroutine/task just created will also be destroyed.